src/modules/0.0.19/aws_ecs_simplified/sql/create_triggers.sql

```diff
@@ -0,0 +1,320 @@
+-- ######################
+-- INSERT ECS SIMPLIFIED
+-- ######################
+
+CREATE OR REPLACE FUNCTION get_mem_from_cpu_mem_enum(cpu_mem ecs_simplified_cpu_mem_enum)
+RETURNS INTEGER LANGUAGE plpgsql AS $$
+DECLARE
+ mem TEXT;
+BEGIN
+  SELECT SPLIT_PART(SPLIT_PART(cpu_mem::TEXT, '-', 2), 'GB', 1) INTO mem;
+  RETURN mem::NUMERIC * 1024;
+END
+$$;
+
+CREATE OR REPLACE FUNCTION get_cpu_mem_enum_from_parts(cpu INTEGER, mem INTEGER)
+RETURNS TEXT LANGUAGE plpgsql AS $$
+DECLARE
+  _cpu DECIMAL;
+  _mem DECIMAL;
+BEGIN
+  _cpu = ROUND(GREATEST(COALESCE(cpu, 256), 256) / 1024.0, 2);
+  _mem = ROUND(GREATEST(COALESCE(mem, 512), 256) / 1024.0, 2);
```

Comment:
nit: should this be
```suggestion
  _mem = ROUND(GREATEST(COALESCE(mem, 512), 512) / 1024.0, 2);
```
?

Reply:
that is a typo, yes